themes/default/content/blog/announcing-kubernetes-helm-release-resource-preview/index.md

```diff
@@ -0,0 +1,305 @@
+	})
+}
+
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+import pulumi
+from pulumi import Output
+from pulumi_random.random_password import RandomPassword
+from pulumi_kubernetes.core.v1 import Namespace, Service
+from pulumi_kubernetes.helm.v3 import Release, ReleaseArgs, RepositoryOptsArgs
+
+namespace = Namespace("redis-ns")
+
+redis_password = RandomPassword("pass", length=10)
+release = Release("redis-helm",
+                  args=ReleaseArgs(
+                      chart="redis",
+                      repository_opts=RepositoryOptsArgs(
+                          repo="https://charts.bitnami.com/bitnami"
+                      ),
+                      version="13.0.0",
+                      namespace=namespace.metadata["name"],
+
+                      # Values from Chart's parameters specified hierarchically,
+                      # see https://artifacthub.io/packages/helm/bitnami/redis/13.0.0#parameters
+                      # for reference.
+                      values={
+                          "cluster": {
+                              "enabled": True,
+                              "slaveCount": 3,
+                          },
+                          "metrics": {
+                              "enabled": True,
+                              "service": {
+                                  "annotations": {
+                                      "prometheus.io/port": "9127",
+                                  }
+                              },
+                          },
+                          "global": {
+                              "redis": {
+                                  "password": redis_password,
+                              }
+                          },
+                          "rbac": {
+                              "create": True,
+                          },
+                      },
+                  ),
+                  # By default Release resource will wait till all created resources
+                  # are available. Set this to true to skip waiting on resources being
+                  # available.
+                  skip_await=False)
+
+# We can lookup resources once the release is installed. The releases'
+# status field is set once the installation completes, so this, combined
+# with `skip_await=False` above, will wait to retrieve the Redis master
+# ClusterIP till all resources in the Chart are available.
+status = release.status
+srv = Service.get("redis-master-svc",
+                  Output.concat(status.namespace, "/", status.name, "-redis-master"))
+pulumi.export("redisMasterClusterIP", srv.spec.cluster_ip)
+
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+Users of the existing Helm Chart resource will notice that we retained the hierarchical `values` specification experience and a similar means to configure the repository. The above example also highlights the await and resource lookup experience.
+
+![helm-release](./helm-release.gif)
```

Comment:
```suggestion
![helm-release](helm-release.gif)
```

I think the `./` are unneeded. Also, it would be good to write meta text within the square brackets describing what's shown in the GIF; that helps folks using screen readers to understand what they can't see.